www/front_src/src/Header/PollerMenu/PollerStatusIcon/index.tsx

```diff
@@ -50,13 +50,36 @@ const useStatusStyles = makeStyles<Theme, StyleProps>((theme) => {
   });
 
   return {
+    avatar: {
+      fontSize: theme.typography.body1.fontSize,
+      height: theme.spacing(2),
+      position: 'relative',
+      width: theme.spacing(2),
+      [theme.breakpoints.down(768)]: {
+        height: theme.spacing(1.8),
+        width: theme.spacing(1.8),
+      },
+    },
+    container: {
+      display: 'flex',
+      gap: theme.spacing(0.5),
+      position: 'relative',
+      right: theme.spacing(0.5),
+      [theme.breakpoints.down(768)]: {
+        gap: theme.spacing(0.4),
+        right: theme.spacing(1),
+        top: theme.spacing(0.4),
+      },
```

Comment:
Are you trying to center an element here?

Reply:
No, I translated the small icons near to the big icon